src/lib.rs

```diff
@@ -370,8 +375,8 @@ pub trait Perform {
 
     /// A final character has arrived for a CSI sequence
     ///
-    /// The `ignore` flag indicates that more than two intermediates arrived and
-    /// subsequent characters were ignored.
+    /// The `ignore` flag indicates that either more than two intermediates arrived
+    /// or the number of parameters exceeded MAX_PARAMS, and subsequent characters were ignored.
```

Comment:
Since `MAX_PARAMS` is not a public constant, I'm not sure if we should use it in our documentation.

Maybe just a more unspecific `exceeded the maximum number of allowed parameters` is less confusing and hopefully doesn't send people on a documentation treasure hunt for this magical `MAX_PARAMS` constant.